src/orchard/query_environment.clj

```diff
@@ -0,0 +1,67 @@
+(ns orchard.query-environment
+  (:require [orchard.meta :as m]
+            [orchard.namespace :as ns]))
+
+(defn list-ns
```

Comment:
All defns should be documented, otherwise the users will really struggle to figure out what to do with them.